test/OpenSpaceToolkit/Physics/Environment.test.cpp

```diff
@@ -753,3 +757,144 @@ TEST (OpenSpaceToolkit_Physics_Environment, Test_3)
 }
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+TEST (OpenSpaceToolkit_Physics_Environment, EarthCreationTest)
+{
+
+    using ostk::core::types::Shared ;
+    using ostk::core::types::Integer ;
+    using ostk::core::ctnr::Array ;
+    using ostk::core::fs::Directory ;
+
+    using ostk::physics::time::Scale ;
+    using ostk::physics::time::Instant ;
+    using ostk::physics::time::DateTime ;
+    using ostk::physics::Environment ;
+    using ostk::physics::env::Object ;
+    using ostk::physics::env::obj::celest::Earth ;
+
+    {
+
+        const Instant instant = Instant::DateTime(DateTime(2018, 1, 1, 0, 0, 0), Scale::UTC) ;
+
+        const Array<Shared<Object>> objects =
+        {
+            std::make_shared<Earth>(Earth::EGM84())
+        } ;
+
+        EXPECT_NO_THROW(Environment environment(instant, objects) ;) ;
+
+    }
+
+    // [TBI] Add a test for the relevant parameters of each earth model at different levels of truncations
+
+}
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+TEST (OpenSpaceToolkit_Physics_Environment, SphericalEarthTest)
```

Comment:
Same test as the previous one but with a different Earth model argument correct? If so, should be a additional scope inside the same test function IMO